Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -102,7 +103,7 @@ public class MSTSWagon : TrainCar
         public string InteriorSoundFileName;
         public string Cab3DSoundFileName;
         public float ExternalSoundPassThruPercent = -1;
-        public float WheelRadiusM = 1;          // provide some defaults in case it's missing from the wag
+        public float WheelRadiusM = Me.FromIn(18.0f);          // provide some defaults in case it's missing from the wag
```

Comment:
Similarly, is there any literature justifying 18in as the default train wheel size?

Reply:
An arbitrary value. Wagon wheels could vary in size from approx 10" to 25".

Reply:
See above

Reply:
Done